docs/streams/upgrade-guide.html

```diff
@@ -91,6 +91,29 @@ <h3><a id="streams_api_changes_280" href="#streams_api_changes_280">Streams API
         We extended <code>StreamJoined</code> to include the options <code>withLoggingEnabled()</code> and <code>withLoggingDisabled()</code> in
         <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-689%3A+Extend+%60StreamJoined%60+to+allow+more+store+configs">KIP-689</a>.
     </p>
+    <p>
+        We added two new methods to Kafka Streams, namely <code>addStreamThread()</code> and <code>removeStreamThread()</code> in
+        <a href="https://cwiki.apache.org/confluence/x/FDd4CQ">KIP-663</a>.
+        These enabled adding a removing StreamThreads to running KafkaStreams client.
```

Comment:
Could you please revisit this sentence since it contains multiple issues?

Reply:
These methods have enabled adding and removing StreamThreads to running a KafkaStreams client.